tests/src/test/scala/whisk/core/cli/test/WskWebActionsTests.scala

```diff
@@ -210,27 +210,6 @@ trait WskWebActionsTests
             authorizedResponse.body.asString shouldBe namespace
     }
 
-    if (testRoutePath == "/api/v1/web") {
```

Comment:
Hmm. This integration test was the test we needed in the first place (and we had a unit test). I don't think we can replace it since the unit tests don't excercise the full route matching.